mono_repo/lib/src/commands/pub.dart

```diff
@@ -133,3 +133,15 @@ final String _sdkDir = (() {
 
 final String pubPath =
     p.join(_sdkDir, 'bin', Platform.isWindows ? 'pub.bat' : 'pub');
+
+/// The path to the root directory of the Flutter SDK.
+/// Requires the "FLUTTER_HOME" environment variable to be set.
+final String _flutterHome = (() {
+  if (!Platform.environment.containsKey('FLUTTER_HOME')) {
+    return '';
+  }
+  return Platform.environment['FLUTTER_HOME']!;
+})();
+
+final String flutterPath =
```

Comment:
Make this private. It's not needed outside this repo!

Reply:
Shall I also make the pubPath variable private while I'm at it?

Reply:
If it's not used anywhere else, yup.